src/matrix/inversewishart.jl

```diff
@@ -43,6 +43,16 @@ size(d::InverseWishart) = (p = dim(d); (p, p))
 params(d::InverseWishart) = (d.df, d.Ψ, d.c0)
 @inline partype{T<:Real}(d::InverseWishart{T}) = T
 
+### Conversion
+function convert{T<:Real}(::Type{InverseWishart{T}}, d::InverseWishart)
+    P = convert_eltype(T, d.Ψ)
```

Comment:
Well, if we leave in `convert_eltype`, I'll simply alias to `convert` for `ZeroVector` and `AbstractArray`. If we update `PDMats`, then that can be aliased, too. 

Happy to go either way you think is best.